internal/component/configmap/etcdconfig.go

```diff
@@ -126,6 +126,12 @@ func prepareInitialCluster(etcd *druidv1alpha1.Etcd, peerScheme string) string {
 		podName := druidv1alpha1.GetOrdinalPodName(etcd.ObjectMeta, i)
 		builder.WriteString(fmt.Sprintf("%s=%s://%s.%s:%s,", podName, peerScheme, podName, domainName, serverPort))
 	}
+
+	if etcd.Spec.Etcd.ExistingCluster != nil {
+		for memberName, memberURLs := range etcd.Spec.Etcd.ExistingCluster.MemberURLs {
+			builder.WriteString(fmt.Sprintf("%s=%s,", memberName, strings.Trim(strings.Join(memberURLs, ","), ",")))
```

Comment:
I think we need some guard rails around this, like validation to ensure that the cluster member names and endpoints are actually valid. I would not want the existing cluster to go into an irrecoverable or unusable state due to bad configuration.

Reply:
And also a check to ensure that the existing cluster is actually up and running. If not, then druid will end up trying to add members to a non-existing or non-running cluster, and go into a futile loop.